src/test/nodestore/varint_test.cpp

```diff
@@ -35,19 +35,14 @@ class varint_test : public beast::unit_test::suite
         testcase("encode, decode");
         for (auto const v : vv)
         {
-            std::array<std::uint8_t,
-                varint_traits<
-                    std::size_t>::max> vi;
-            auto const n0 =
-                write_varint(
-                    vi.data(), v);
+            std::array<std::uint8_t, varint_traits<std::size_t>::max> vi;
+            auto const n0 = write_varint(vi.data(), v);
             expect (n0 > 0, "write error");
             std::size_t v1;
-            auto const n1 =
-                read_varint(
-                    vi.data(), n0, v1);
+            auto const n1 = read_varint(vi.data(), n0, v1);
             expect(n1 == n0, "read error");
             expect(v == v1, "wrong value");
+            expect(n0 == size_varint(v), "size error");
```

Comment:
Might be better placed after line 40 or replace that expect.

Reply:
Moved, thanks.